lib/SiftClient.php

```diff
@@ -16,6 +16,25 @@ class SiftClient {
     private $timeout;
     private $version;
 
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @param \Psr\Log\LoggerInterface $logger
+     */
+    public function setLogger($logger)
+    {
+        $this->logger = $logger;
+    }
+
+    private function logError($message, array $context = array()) {
+        if ($this->logger instanceof \Psr\Log\LoggerInterface){
```

Comment:
With the signature you don't need to check instance of. Only `if ($this->logger)`